python/python/lancedb/table.py

```diff
@@ -936,21 +933,38 @@ def cleanup_old_versions(
         CleanupStats
             The stats of the cleanup operation, including how many bytes were
             freed.
+
+        See Also
+        --------
+        [Table.optimize][lancedb.table.Table.optimize]: A more comprehensive
+            optimization operation that includes cleanup as well as other operations.
+
+        Notes
+        -----
+        This function is not available in LanceDb Cloud (since LanceDB
+        Cloud manages cleanup for you automatically)
         """
 
     @abstractmethod
     def compact_files(self, *args, **kwargs):
         """
         Run the compaction process on the table.
-
-        Note: This function is not available in LanceDb Cloud (since LanceDb
-        Cloud manages compaction for you automatically)
-
         This can be run after making several small appends to optimize the table
         for faster reads.
 
-        Arguments are passed onto :meth:`lance.dataset.DatasetOptimizer.compact_files`.
+        Arguments are passed onto Lance's
+        [compact_files][lance.dataset.DatasetOptimizer.compact_files].
         For most cases, the default should be fine.
+
+        See Also
+        --------
+        [Table.optimize][lancedb.table.Table.optimize]: A more comprehensive
+            optimization operation that includes cleanup as well as other operations.
+
+        Notes
+        -----
+        This function is not available in LanceDB Cloud (since LanceDB
```

Comment:
If we make the remote methods no-ops instead of raising an error then we just remove this?

Reply:
I think we can at least say it's unnecessary in the docs.